src/openzaak/components/documenten/tests/management/test_delete_empty_eioc.py

```diff
@@ -0,0 +1,68 @@
+# SPDX-License-Identifier: EUPL-1.2
+# Copyright (C) 2024 Dimpact
+
+from django.core.management import call_command
+
+from rest_framework import status
+from rest_framework.test import APITestCase
+from vng_api_common.tests import reverse
+
+from openzaak.components.besluiten.models import BesluitInformatieObject
+from openzaak.components.besluiten.tests.factories import (
+    BesluitFactory,
+    BesluitInformatieObjectFactory,
+)
+from openzaak.components.zaken.models import ZaakInformatieObject
+from openzaak.components.zaken.tests.factories import (
+    ZaakFactory,
+    ZaakInformatieObjectFactory,
+)
+from openzaak.components.zaken.tests.utils import ZAAK_WRITE_KWARGS
+from openzaak.tests.utils import JWTAuthMixin
+
+from ...models import EnkelvoudigInformatieObjectCanonical
+from ..factories import EnkelvoudigInformatieObjectCanonicalFactory
+
+
+class DeleteEmptyEIOCTests(JWTAuthMixin, APITestCase):
+    heeft_alle_autorisaties = True
+
+    def setUp(self):
+        super().setUp()
+        self.client.raise_request_exception = False
```

Comment:
If the tests pass, the issue was that with DEBUG set to true, it will raise an exception instead of returning a 500 error